dark1/memberavatarstatus/core/memberavatarstatus.php

```diff
@@ -30,7 +30,7 @@ class memberavatarstatus
 	const AV_MIN_SZ = 9;
```

Comment:
constants should go preferably into the ext.php file.

Then `use vendor/name/ext;`

Reply:
I was told by extension validation team to put it in core file ie `./core/memberavatarstatus.php` .